vagga.yaml

```diff
@@ -3,11 +3,13 @@ containers:
   pip-freeze:
     setup:
     - !Alpine v3.4
+    - !BuildDeps [postgresql-dev]
     - !Py3Install
       - pip
       - 'Django >=1.8, <2.0'
       - 'gunicorn >=19.6.0, <20.0'
       - 'whitenoise >=3.2.3, <4.0'
+      - 'psycopg2 >=2.6.2, <3.0'
       - 'django-environ >=0.4.1, <1.0'
```

Comment:
Leave only by minor version. Patch version is not needed here.
Also [read](http://semver.org/#semantic-versioning-200) this, please